### ISSUE TITLE
Build fails when space in path

### ISSUE THREAD

**Issue by Cactucs** [lovell/sharp#533]

**Comment by lovell** [comment]
Hello, is this on Windows? #488

**Comment by Cactucs** [comment]
Ubuntu 14.04

**Comment by lovell** [comment]
Ah, I've just reproduced this. I wonder if something has changed in gyp/node-gyp recently? It looks like https://github.com/nodejs/node-gyp/issues/65 might still be the underlying problem as `module_root_dir` doesn't appear to be correctly escaped. Perhaps we can work around this - leave it with me.

**Comment by lovell** [comment]
Commit da0dc28 on the `pencil` branch appears to have solved this:

````sh
mkdir "/tmp/hello world" && cd "/tmp/hello world" && npm install lovell/sharp#pencil
```

More importantly it hasn't broken anything else on [Linux](https://travis-ci.org/lovell/sharp/jobs/148866282), [OS X](https://travis-ci.org/lovell/sharp/jobs/148866283) and [Windows](https://ci.appveyor.com/project/lovell/sharp/build/597/job/3tam4cqywlxwfos2).

Thanks for reporting this.

**Comment by lovell** [comment]
This should be fixed in v0.16.0, now available via npm.
